test/dataframe.jl

```diff
@@ -350,7 +346,7 @@ module TestDataFrame
                                                           :string, :string_missing,
                                                           :dates, :catarray],
                                               mean = [2.5, 2.0, nothing, nothing, nothing, nothing],
-                                              std = [Compat.std(df[:number]), 1.0, nothing, 
+                                              std = [Statistics.std(df[:number]), 1.0, nothing, 
```

Comment:
No need to prefix with the module, right?

Reply:
Removed `Statistics.`